lib/Service/SampleConversationsService.php

```diff
@@ -114,7 +114,7 @@ protected function defaultSampleConversation(IUser $user): void {
 			. "\n\n" . '{FILE:Readme.md}',
 			$this->l->t('## 💭 Let the conversations flow: mention users, react to messages and more
 
-You can mention everybody in a the conversation @all or mention specific participants by typing "@" and picking their name from the list.')
+You can mention everybody in a the conversation "@all" or mention specific participants by typing "@" and picking their name from the list.')
```

Comment:
This was intentional to should the mention chip. If it's a problem to translations I will make it a parameter
```suggestion
You can mention everybody in a the conversation @all or mention specific participants by typing "@" and picking their name from the list.')
```